test/menu-bar.test.js

```diff
@@ -319,9 +322,33 @@ describe('overflow button', () => {
   });
 
   it('should hide overflow button and reset its items when all buttons fit', async () => {
-    menu.style.width = 'auto';
+    container.style.width = '500px';
     menu.notifyResize();
-    await nextRender(menu);
+    await nextRender(container);
+    assertVisible(buttons[2]);
+    expect(buttons[2].disabled).to.not.be.true;
+    assertVisible(buttons[3]);
+    expect(buttons[3].disabled).to.not.be.true;
+    assertVisible(buttons[4]);
+    expect(buttons[4].disabled).to.be.true;
+    expect(overflow.hasAttribute('hidden')).to.be.true;
+    expect(overflow.item.children.length).to.equal(0);
+  });
+
+  it('should hide overflow button and reset its items when all buttons fit even with min-width set to 0', async () => {
```

Comment:
The test case name (and the PR title) is a bit too technical about the preconditions under which the bug occurs. You can also make it happen without a flexbox container by using for example one of the following on the menu-bar element:
- `width: max-content; max-width: 100%;`
- `display: inline-block; max-width: 100%;`
- `position: absolute; max-width: 100%;`

...meaning that it's not specifically about `min-width: 0`.

Note: you could avoid changes to the test fixture by using the original fixture and applying one of these styles programmatically to the element at the beginning of the test.

Reply:
I will move the test to a new fixture because the menu-bar has to be in a container (e.g.: a div element) to reproduce the bug.